packages/dx-react-chart/demos/pie/doughnut.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Doughnut
+
+The demo shows [Doughnut Chart](../../docs/reference/pie-series.md).
```

Comment:
This reference is unclear to me. Maybe we should write `This demo shows Doughnut Chart. You can specify the PieSeries[ref] property 'innerRadius' to achieve this behaver`

Reply:
Ok